src/core/QCheck.mli

```diff
@@ -397,6 +397,26 @@ module Gen : sig
       Example: [char_range 'a' 'z'] for all lower case ascii letters.
       @since 0.13 *)
 
+  val bytes_size : ?gen:char t -> int t -> bytes t
+  (** Builds a bytes generator from a (non-negative) size generator.
+      Accepts an optional character generator (the default is {!char}). *)
+
+  val bytes : ?gen:char t -> bytes t
+  (** Builds a bytes generator. Bytes size is generated by {!nat}.
+      Accepts an optional character generator (the default is {!char}).
+      See also {!bytes_of} and {!bytes_readable} for versions without
+      optional parameters. *)
+
+  val bytes_of : char t -> bytes t
+  (** Builds a bytes generator using the given character generator. *)
+
+  val bytes_printable : bytes t
+  (** Builds a bytes generator using the {!printable} character generator. *)
+
+  val small_bytes : ?gen:char t -> bytes t
+  (** Builds a bytes generator, length is {!small_nat}
+      Accepts an optional character generator (the default is {!char}). *)
```

Comment:
Should we add the tag `@since NEXT_RELEASE` as well as updating the changelog for the new generators?